cli/internal/jsonmessage/display.go

```diff
@@ -0,0 +1,32 @@
+package jsonmessage
+
+import (
+	"context"
+	"io"
+
+	"github.com/docker/docker/pkg/jsonmessage"
+)
+
+type (
+	Stream      = jsonmessage.Stream
+	JSONMessage = jsonmessage.JSONMessage
+	JSONError   = jsonmessage.JSONError
```

Comment:
This is still an `internal`  package, so not too problematic, but it looks like `JSONError` is literally used in a single place in the CLI. We may want to have a look if we want to keep it  as an exported type, or if there's alternatives to reduce the "public" (but right now, still "internal") API; https://github.com/docker/cli/blob/f4a68da19595d64c50b0bbc2b1f15e645943ed82/cli/command/image/build.go#L366-L375

Reply:
The idea behind this was to keep all the `jsonmessage` code in a singular place inside and internal package inside the CLI. Maybe i'm being a bit excessive, but it _felt_ cleaner